js/src/ui/ConfirmDialog/confirmDialog.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react';
+import ActionDone from 'material-ui/svg-icons/action/done';
+import ContentClear from 'material-ui/svg-icons/content/clear';
+
+import Button from '../Button';
+import Modal from '../Modal';
+
+import styles from './confirmDialog.css';
+
+export default class ConfirmDialog extends Component {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+    iconNo: PropTypes.node,
+    iconYes: PropTypes.node,
+    labelNo: PropTypes.string,
+    labelYes: PropTypes.string,
+    title: PropTypes.oneOfType([
+      PropTypes.node, PropTypes.string
+    ]).isRequired,
+    visible: PropTypes.bool.isRequired,
+    onNo: PropTypes.func.isRequired,
+    onYes: PropTypes.func.isRequired
```

Comment:
I'd prefer `onConfirm` and something like `onReject` or `onDeny` here. It is more expressive than "yes" and "no". Same for `labelYes` and `labelNo`.